project-base/templates/Front/Content/Category/panel.html.twig

```diff
@@ -5,7 +5,7 @@
         {% if categoriesWithLazyLoadedVisibleChildren[0].category.level > 2 %} list-menu--dropdown{% endif %}"
         {% if isFirstLevel %}id="js-categories"{% endif %}>
         {% for categoryWithLazyLoadedVisibleChildren in categoriesWithLazyLoadedVisibleChildren %}
-            {% set isCurrentCategory = (currentCategory is not null and currentCategory == categoryWithLazyLoadedVisibleChildren.category) %}
+            {% set isCurrentCategory = (currentCategory is not null and currentCategory is same as (categoryWithLazyLoadedVisibleChildren.category)) %}
```

Comment:
are those parentheses around `categoryWithLazyLoadedVisibleChildren.category` necessary?
For me, it does not improve readability.

Reply:
That is right see https://twig.symfony.com/doc/2.x/tests/sameas.html

I have removed space between as and "("